src/main/java/org/example/marketplacebackend/repository/WatchListRepository.java

```diff
@@ -1,5 +1,13 @@
 package org.example.marketplacebackend.repository;
 
-public interface WatchListRepository {
+import org.example.marketplacebackend.model.Account;
+import org.example.marketplacebackend.model.ProductCategory;
+import org.example.marketplacebackend.model.Watchlist;
+import org.springframework.data.jpa.repository.JpaRepository;
+import java.util.List;
+import java.util.UUID;
 
-}
+public interface WatchListRepository extends JpaRepository<Watchlist, UUID> {
+  List<Watchlist> findAllBySubscriber(Account subscriber);
+  void deleteBySubscriberAndProductCategory(Account subscriber, ProductCategory productCategoryID);
```

Comment:
Change the void return type to Long to get access to the number of affected rows (related to feedback in the delete endpoint).